plugins/node/opentelemetry-instrumentation-aws-sdk/test/dynamodb.test.ts

```diff
@@ -81,6 +88,102 @@ describe('DynamoDB', () => {
           );
           expect(attrs[SemanticAttributes.DB_NAME]).toStrictEqual('test-table');
           expect(attrs[SemanticAttributes.DB_OPERATION]).toStrictEqual('Query');
+          expect(
```

Comment:
generally about the tests: I see not all the attributes are tested. can we add the missing ones? 
AWS_DYNAMODB_PROVISIONED_READ_CAPACITY
AWS_DYNAMODB_PROVISIONED_WRITE_CAPACITY
AWS_DYNAMODB_GLOBAL_SECONDARY_INDEXES
AWS_DYNAMODB_LOCAL_SECONDARY_INDEXES
AWS_DYNAMODB_EXCLUSIVE_START_TABLE
AWS_DYNAMODB_ATTRIBUTE_DEFINITIONS
AWS_DYNAMODB_GLOBAL_SECONDARY_INDEX_UPDATES
AWS_DYNAMODB_ITEM_COLLECTION_METRICS
AWS_DYNAMODB_TABLE_COUNT

Reply:
thanks! I see lint is failing. can you run `npm run lint:fix` to fix this? then we can approve this. thanks!!!